src/bw_admin.rs

```diff
@@ -31,29 +33,50 @@ impl User {
 pub struct Client {
     url: String,
     admin_token: String,
-    cookie: Option<String>,
-    cookie_created: Option<Instant>,
+    root_cert: String,
+	cookie: Option<String>,
+    cookie_created: Option<Instant>
```

Comment:
Indentation appears to be off a bit here. Please run `rustfmt` on everything. I'll add a pre-commit hook to automate this in the future.

There are other similar formatting issues that I will not mark since `rustfmt` should take care of them.

Reply:
It had tabs instead of spaces at a few places. I ran `rustfmt` on all source files so formatting should be fine now.